x-pack/solutions/security/plugins/cloud_security_posture/server/routes/graph/fetch_graph.ts

```diff
@@ -187,9 +189,9 @@ ${
     "\\"id\\":\\"", actor.entity.id, "\\"",
     ",\\"type\\":\\"", "${DOCUMENT_TYPE_ENTITY}", "\\"",
     ",\\"entity\\":", "{",
-      "\\"name\\":\\"", actorEntityName, "\\"",
-      ",\\"type\\":\\"", actorEntityType, "\\"",
-      ",\\"sub_type\\":\\"", actorEntitySubType, "\\"",
+      "\\"name\\":\\"", COALESCE(actorEntityName, ""), "\\"",
```

Comment:
I think that instead of having empty strings it should be `undefined`. This way it will make it clearer that the text is missing.